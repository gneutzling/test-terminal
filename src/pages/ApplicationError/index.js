import React from "react"

const ApplicationError = () => {
  return <h1>ApplicationError</h1>
}

export default ApplicationError
