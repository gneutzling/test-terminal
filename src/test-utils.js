import React from "react"
import { render } from "@testing-library/react"
import "jest-dom/extend-expect"

import RouterProvider from "app/lib/RouterProvider"

const AllTheProviders = ({ children }) => {
  return <RouterProvider>{children}</RouterProvider>
}

const customRender = (ui, options) =>
  render(ui, { wrapper: AllTheProviders, ...options })

// re-export everything
export * from "@testing-library/react"

// override render method
export { customRender as render }
