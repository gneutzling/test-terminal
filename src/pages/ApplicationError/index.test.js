import React from "react"
import { render } from "app/test-utils"
import ApplicationError from "./"

describe("ApplicationError", () => {
  it("matches snapshot", () => {
    const { container } = render(<ApplicationError />)
    expect(container).toMatchSnapshot()
  })
})
