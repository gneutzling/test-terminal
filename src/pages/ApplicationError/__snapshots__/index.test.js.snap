// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationError matches snapshot 1`] = `
<div>
  <h1>
    ApplicationError
  </h1>
</div>
`;
